modules/cbQuestion/cbQuestionWidget.php

```diff
@@ -43,7 +43,7 @@ public function process($context = false) {
 				'$MODULE$' => $module,
 				'$USERID$' => $current_user->id,
 			);
-			if (isset($recordid) && $recordid != '0x0') {
+			if (isset($recordid) && $recordid != '0x0' && !empty($recordid)) {
```

Comment:
should be

`if (!empty($recordid) && $recordid!='0x0') {`

because if the variable is !empty include isset  (I think)

Reply:
I think isset checks if it is declared .. not if it is empty
![image](https://user-images.githubusercontent.com/59079190/204742431-520890a1-0f20-4f22-92d7-9ca39258c149.png)